src/swtpm/capabilities.c

```diff
@@ -122,19 +123,28 @@ int capabilities_print_json(bool cusetpm)
 #else
     const char *cmdarg_seccomp = "";
 #endif
+    const char *with_tpm1 = "";
+    const char *with_tpm2 = "";
     char *keysizecaps = NULL;
 
     ret = get_rsa_keysize_caps(&keysizecaps);
     if (ret < 0)
         goto cleanup;
 
+    if (TPMLIB_ChooseTPMVersion(TPMLIB_TPM_VERSION_1_2) == TPM_SUCCESS)
+        with_tpm1 = "\"tpm-1.2\", ";
+    if (TPMLIB_ChooseTPMVersion(TPMLIB_TPM_VERSION_2) == TPM_SUCCESS)
+        with_tpm2 = "\"tpm-2.0\", ";
```

Comment:
Same here, or we show both, no-tpm-2.0 and tpm-2.0, which would make it clear whether it is available or not.

Reply:
test_tpm2_print_capabilities and test_print_capabilities will need to be adjusted as well..

Reply:
I don't think it's necessary. (see my other comment)